ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/tabs/clientlibs/less/tabs.less

```diff
@@ -0,0 +1,236 @@
+/* tabs style */
```

Comment:
@rakeshkumar-verma @Rupesh-Shende : whys is font color grey on grey bg ? is it as per UX ..

![image](https://user-images.githubusercontent.com/38690824/81677732-d17a7b80-946e-11ea-9a1c-f923b402ae2e.png)

Reply:
@rishabh-behl Yes, It is as per UX shared by Ashish.
https://hclo365-my.sharepoint.com/personal/rishabh_b_hcl_com1/_layouts/15/onedrive.aspx?originalPath=aHR0cHM6Ly9oY2xvMzY1LW15LnNoYXJlcG9pbnQuY29tLzpmOi9nL3BlcnNvbmFsL3Jpc2hhYmhfYl9oY2xfY29tMS9FcVdjQUJ4RURkVk9ycU9xUjdTaG9ud0JqM1YzVmROMTcxMUlMX28zN3Npb2N3P3J0aW1lPUIwQXp6UVBoMTBn&id=%2Fpersonal%2Frishabh%5Fb%5Fhcl%5Fcom1%2FDocuments%2FAdobe%2FHCL%2DeCommerce%2FUI%2DUX%2FDesigns%2FTab%2Ftabbing%5F01%2Ejpg&parent=%2Fpersonal%2Frishabh%5Fb%5Fhcl%5Fcom1%2FDocuments%2FAdobe%2FHCL%2DeCommerce%2FUI%2DUX%2FDesigns%2FTab